test/api.js

```diff
@@ -0,0 +1,29 @@
+import events from './data/events';
+import projects from './data/projects';
```

Comment:
Nit: Filename should probably be `productions`, not `projects`?

Reply:
yeah definitely